src/scene/lightmap.js

```diff
@@ -0,0 +1,271 @@
+pc.extend(pc.Application.prototype, function () {
```

Comment:
It looks like you need app for getting the model list. Maybe better to pass in a list of models to bake?